app/components/ui/molecules/PeoplePicker.test.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import { ThemeProvider } from 'styled-components'
+import theme from '@elifesciences/elife-theme'
+import PeoplePicker from './PeoplePicker'
+
+const people = [
+  { id: 1, name: 'Annie' },
+  { id: 2, name: 'Bobby' },
+  { id: 3, name: 'Chastity' },
+  { id: 4, name: 'Dave' },
+]
+
+const makeWrapper = props =>
+  mount(
+    <ThemeProvider theme={theme}>
+      <PeoplePicker
+        // eslint-disable-next-line react/no-children-prop
+        children={innerProps => <PeoplePicker.Body {...innerProps} />}
+        onCancel={jest.fn()}
+        onSubmit={jest.fn()}
+        people={people}
+        {...props}
+      />
+    </ThemeProvider>,
+  )
+
+const getPersonPodButton = (wrapper, index) =>
+  wrapper.find('button[data-test-id="person-pod-toggle"]').at(index)
+
+const expectSelectionLength = (wrapper, length) =>
+  expect(wrapper.find('SelectedItem')).toHaveLength(length)
+
+describe('PeoplePicker', () => {
+  it('calls child render prop', () => {
+    const render = jest.fn(() => null)
+    makeWrapper({ children: render })
+    expect(render).toHaveBeenCalled()
+  })
+
+  it('renders one person pod per person', () => {
+    const wrapper = makeWrapper()
+    const buttons = wrapper.find('PersonPod')
+
+    expect(buttons).toHaveLength(4)
+  })
+
+  it('updates selection', () => {
+    const wrapper = makeWrapper()
+    expectSelectionLength(wrapper, 0)
+
+    getPersonPodButton(wrapper, 0).simulate('click')
+    expectSelectionLength(wrapper, 1)
+
+    getPersonPodButton(wrapper, 0).simulate('click')
+    expectSelectionLength(wrapper, 0)
+  })
+
+  it('disables inputs when maximum is reached', () => {
```

Comment:
Similar to my comment above - this test is relying on a lot of things at once, many of which aren't being tested individually. So the description could potentially be misleading.

For example, this test would pass even if the icon on that particular pod was changing to a Tick, so long as the selection array wasn't being updated & there was no `SelectedItem` on the page.

So the description of this test would be a bit of a false friend, as the "inputs" (button clicks?) could theoretically still be enabled & the icons still changing accordingly.